gallery/src/components/NxStatefulTreeViewRadioSelect/NxStatefulTreeViewRadioSelectPage.tsx

```diff
@@ -22,7 +22,10 @@ const NxStatefulTreeViewRadioSelectPage = () =>
         to <NxCode>NxStatefulCollapsibleRadioSelect</NxCode> and
         is now deprecated and will be removed in the next major version.
         Please use
-        {' '}<NxTextLink href="#/pages/NxStatefulCollapsibleRadioSelect">NxStatefulCollapsibleRadioSelect</NxTextLink>
+        {' '}
```

Comment:
Minor: move this to previous line

Reply:
Fixed: [8e9fa45](https://github.com/sonatype/sonatype-react-shared-components/pull/721/commits/8e9fa45236f867c20f079aaaacaf49c4db324b9e)